services/QuillLMS/lib/tasks/flagsets.rake

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'csv'
+
+namespace :flagsets do
+  namespace :users do
+    # Example usage: rake 'flagsets:users:update_flagsets_from_csv[filepath, flagset]'
+    desc 'Add flagsets to users from csv'
+    task :update_flagsets_from_csv, [:filepath, :flagset] => :environment do |t, args|
+      iostream = File.read(args[:filepath])
+      if (CSV.parse(iostream, headers: true).headers & ["email", "flagset"]).count != 2
```

Comment:
Seems a little odd to require a `flagset` header if we also override that value if we pass in `flagset` as an arg, but as long as we document the behavior it should be fine since this is a purely internal tool.

Reply:
It's for flexibility - sometimes Curriculum will want a mix of flagsets uploaded. Sometimes they just give a list of emails. In the later case, it's nice to have the parameter override, so you don't need to populate a CSV column of repetitive values.